aptos-move/move-examples/marketplace/sources/coin_listing.move

```diff
@@ -0,0 +1,1080 @@
+address marketplace {
```

Comment:
Are you experimenting with this for the test module?

Reply:
ahh... so the separate module is great because otherwise you end up with a ton of #[test_only] that aren't readable. same file is great because you know exactly where the demo code is.